### PR TITLE
Do not ship policy.yaml by default

### PR DESCRIPTION
**Pull request by fmount**
`policy.yaml` is stored in a secret at the moment: it can't be changed by any human operator because `glance-operator` will restore it and it grows a the size of the `GlanceAPI` secret.
This patch removes it from the `Secret`, `kolla` won't sync it by default but it can still be provided via `extraVolumes`, and the `oslo_policy` section can be injected via `customConfigService` overrides.

### CODE REVIEW

File: config/samples/policy/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+[example](https://github.com/openstack-k8s-operators/glance-operator/tree/main/config/samples/glance_v1beta_glance_apply_policy.yaml),
+when the `ConfigMap` is available, a `GlanceAPI` CR can be applied injecting
+the configuration that will point to the new `policy.yaml` file.
+
+```
+spec:
+  ...
+  customServiceConfig: |
+    [oslo_policy]
+    policy_file=/etc/glance/policy.d/policy.yaml
```
Comment (konan-abhi) [inline config/samples/policy/README.md:31]: Shall we also mention about 'enforce_scope' and 'enforce_new_defaults' parameters?
Those are set to true if SRBAC needs to be enabled.

https://github.com/openstack/oslo.policy/blob/master/oslo_policy/opts.py#L27

**Review by konan-abhi — COMMENTED** [review]
One suggestion, otherwise looks perfect

File: config/samples/policy/README.md (hunk 1 above)
Comment (fmount) [reply config/samples/policy/README.md:31]: of course, good catch! updating the patch in a bit!

**Review by konan-abhi — APPROVED** [review]
Looks perfect, thank you!

**Pull request merged**